### PR TITLE
Add `scipy.stats.qmc` tutorial

### PR DESCRIPTION
**Pull request by tupui**

### CODE REVIEW

**Comment by jarrodmillman** [comment]
This looks great, but I noticed something I need to fix in the theme. I will look into it this morning and then we can test it with this PR.

**Comment by tupui** [comment]
@jarrodmillman are you talking about inline-maths? I am seeing that `$$...$$` works fine but not `$...$`

**Comment by jarrodmillman** [comment]
I also want to try using mathjax 3.

File: content/posts/scipy/qmc-basics/index.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+A very similar interface, but as seen bellow, with radically different results.
+
+![Comparision between MC and QMC samples using a 2D scatter plot.](mc_sobol.png)
+
+The 2D space clearly exhibit less empty areas and less clusters with the QMC
+sample.
+
+## Quality?
+
+Past the visual improvement of _quality_, there are metrics to assess the
```
Comment (jarrodmillman) [inline content/posts/scipy/qmc-basics/index.md:135]:
```suggestion
Beyond the visual improvement of _quality_, there are metrics to assess the
```

File: content/posts/scipy/qmc-basics/index.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+```
+
+![Convergence of the integration error with MC and QMC.](conv_mc_sobol.png)
+
+With MC the approximation error follows a theoretical rate of $O(n^{-1/2})$.
+But, QMC methods have better rates of convergence and achieve $O(n^{-1})$
+for this function–and even better rates on very smooth functions.
+
+This means that using $2^8=256$ points from _Sobol'_ leads to a lower
+error than using $2^12=4096$ points from MC! When the function evaluation
```
Comment (jarrodmillman) [inline content/posts/scipy/qmc-basics/index.md:243]:
```suggestion
error than using $2^{12}=4096$ points from MC! When the function evaluation
```

File: content/posts/scipy/qmc-basics/index.md [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+is costly, it can bring huge computational savings.
+
+## Sampling from any distribution (advanced)
+
+But there is more! Another great use of QMC is to sample arbitrary
+distributions. In SciPy 1.8, there are new classes of
+[samplers](https://scipy.github.io/devdocs/reference/stats.sampling.html)
+that allow to sample from any custom distribution. And it happens that some of
```
Comment (jarrodmillman) [inline content/posts/scipy/qmc-basics/index.md:251]:
```suggestion
that allow you to sample from any custom distribution. And it happens that some of
```

Comment (jarrodmillman) [inline content/posts/scipy/qmc-basics/index.md:251]:
```suggestion
that allow to sample from any custom distribution. And some of
```

**Comment by jarrodmillman** [comment]
LGTM.  I added some minor suggestions/fixes.

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**